libqtile/widget/check_updates.py

```diff
@@ -78,3 +79,8 @@ def poll(self):
         if not self.cmd:
             return "N/A"
         return self._check_updates()
+
+    def button_press(self, x, y, button):
+        base.ThreadedPollText.button_press(self, x, y, button)
+        if button == 1 and self.execute is not None:
+            Popen([self.execute], shell=True)
```

Comment:
You might make this just self.execute here, so that people can pass in a list if their distro update command (i.e. `apt-get deselect-upgrade`) is more than one word.